backend/apps/sandbox/views.py

```
@@ -8,13 +8,17 @@
 from .services import verify_git_command
 
 
+from rest_framework.throttling import ScopedRateThrottle
+
 class SandboxVerifySerializer(serializers.Serializer):
     command = serializers.CharField()
     expected_command = serializers.CharField()
 
 
 class SandboxVerifyView(APIView):
     permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)